tsl/test/isolation/expected/dist_restore_point.out

```diff
@@ -0,0 +1,333 @@
+Parsed test spec with 3 sessions
+
+starting permutation: s2_setup
+node_name      host           port           database       node_created   database_createdextension_created
+
+data_node_1    localhost      55432          cdrp_1         t              t              t              
+node_name      host           port           database       node_created   database_createdextension_created
+
+data_node_2    localhost      55432          cdrp_2         t              t              t              
+node_name      host           port           database       node_created   database_createdextension_created
+
+data_node_3    localhost      55432          cdrp_3         t              t              t              
+step s2_setup: SELECT created FROM create_distributed_hypertable('disttable', 'time', 'device');
+created        
+
+t              
+
+starting permutation: s3_lock_enable s1_create_dist_rp s2_insert s3_lock_count s3_lock_release
+node_name      host           port           database       node_created   database_createdextension_created
+
+data_node_1    localhost      55432          cdrp_1         f              f              f              
+node_name      host           port           database       node_created   database_createdextension_created
+
+data_node_2    localhost      55432          cdrp_2         f              f              f              
+node_name      host           port           database       node_created   database_createdextension_created
+
+data_node_3    localhost      55432          cdrp_3         f              f              f              
+step s3_lock_enable: SELECT debug_waitpoint_enable('create_distributed_restore_point_lock');
+debug_waitpoint_enable
+
+               
+step s1_create_dist_rp: SELECT restore_point > pg_lsn('0/0') as valid_lsn FROM create_distributed_restore_point('s1_test'); <waiting ...>
+step s2_insert: INSERT INTO disttable VALUES ('2019-08-02 10:45', 0, 0.0); <waiting ...>
+step s3_lock_count: 
+	SELECT waitpoint_locks('create_distributed_restore_point_lock') as cdrp_locks, 
+	       remote_txn_locks() as remote_txn_locks;
+
+cdrp_locks     remote_txn_locks
+
+2              2              
```

Comment:
What's expected in terms of lock counts here (maybe add in comment). I'd expect 2 cdrp locks (one for the lock session and one for the create-restore-point session). But why do we see 2 remote_txn locks when there is only one INSERT session?

Would be useful to break this down in waiters and holders (although I guess there is always 1 holder).

Reply:
Because first lock is already taken by the create_distributed_restore_point() function. Basically waitpoint is reached after this lock being taken